theme/ItaliaTheme/_main.scss

```diff
@@ -270,4 +270,44 @@ iframe {
       white-space: nowrap;
     }
   }
-}
\ No newline at end of file
+}
+
+// fix footer line and columns without title
+.it-footer-main {
+  .row > div {
+    h4:empty:before {
+      content: '\200b';
+      @media (max-width: #{map-get($grid-breakpoints, md)}) {
+        content: none;
+      }
+    }
+
+    h4 {
+      margin-bottom: 0.889rem !important;
+      padding-bottom: 0.889rem;
+      border-bottom: 1px solid hsla(0, 0%, 100%, 0.5);
+      @media (max-width: #{map-get($grid-breakpoints, md)}) {
+        &:empty {
+          margin-bottom: 0 !important;
+          padding-bottom: 0 !important;
+        }
+      }
+    }
+  }
+  .row > div ~ div h4:empty {
+    position: relative;
+    border-bottom: none;
+    &::after {
+      content: '';
+      position: absolute;
+      width: calc(100% + 12px);
+      background: hsla(0, 0%, 100%, 0.5);
+      height: 1px;
+      bottom: -1px;
+      left: -12px;
```

Comment:
Qui conviene probabilmente usare `$grid-gutter-width` invece di `12px` in entrambi i punti (`width` e `left`), soprattutto perché con la nuova versione di bootstrap-italia la larghezza del gutter cambia e quindi poi questo lo ritroveremmo rotto